src/annotations/validateAnnotations/AdulthoodValidator.java

```diff
@@ -0,0 +1,31 @@
+package annotations.validateAnnotations;
+
+import annotations.Adulthood;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.time.LocalDate;
+
+
+public class AdulthoodValidator implements Validator {
+
+    @Override
+    public void fieldValidator(Class<?> className) {
+        Field[] declaredFields = className.getDeclaredFields();
+        for (Field field : declaredFields) {
+            if (field.isAnnotationPresent(Adulthood.class)) {
+                field.setAccessible(true);
+                try {
+                    LocalDate fieldValue = (LocalDate) field.get(className.getDeclaredConstructor().newInstance());
+                    if ((LocalDate.now().getYear() - fieldValue.getYear()) <= 18){
+                        if ((LocalDate.now().getDayOfYear() - fieldValue.getDayOfYear()) < 0){
+                            throw new RuntimeException("You are under 18 years old");
```

Comment:
Instead of throwing exception, create a List<String> messages = new ArrayList<>();
and add into that list message about validation. For example in this case it should be, 'You are under 18 years old".
If Dto did not pass multiple validations then at the end you must have a messages list with multiple elements.